src/components/FinalPoem.js

```diff
@@ -1,21 +1,32 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import './FinalPoem.css';
 
 const FinalPoem = (props) => {
+  const [poemPrintOut, setPoemPrintOut] = useState('');
 
+  const revealFinalPoem = () => {
+    if (!props.isSubmitted) {
+      setPoemPrintOut(props.submissions.map(p => <p>{p}</p>));
```

Comment:
You need to add a `key` attribute to this JSL array that `map` is returning. 

```suggestion
      setPoemPrintOut(props.submissions.map((p, i) => <p key={i}>{p}</p>));
```